assets/xml/sequences/Sequences.xml

```diff
@@ -0,0 +1,112 @@
+    <Sequence Name="020_Hyrule_Field_Idle_1"/>
+    <Sequence Name="021_Hyrule_Field_Idle_2"/>
+    <Sequence Name="022_Hyrule_Field_Idle_3"/>
+    <Sequence Name="023_Hyrule_Field_Idle_4"/>
+    <Sequence Name="024_Dodongos_Cavern"/>
+    <Sequence Name="025_Kakariko_Village_Adult"/>
+    <Sequence Name="026_Battle"/>
+    <Sequence Name="027_Boss_Battle"/>
+    <Sequence Name="028_Inside_the_Deku_Tree"/>
+    <Sequence Name="029_Market"/>
+    <Sequence Name="030_Title_Theme"/>
+    <Sequence Name="031_House"/>
+    <Sequence Name="032_Game_Over"/>
+    <Sequence Name="033_Boss_Defeated"/>
+    <Sequence Name="034_Got_Key_Item"/>
+    <Sequence Name="035_Ganondorf_Sting"/>
+    <Sequence Name="036_Got_Heart_Container"/>
+    <Sequence Name="037_Prelude_of_Light"/>
+    <Sequence Name="038_Inside_Jabu_Jabus_Belly"/>
+    <Sequence Name="039_Kakariko_Village_Child"/>
+    <Sequence Name="040_Fairy_Fountain"/>
+    <Sequence Name="041_Zeldas_Theme"/>
+    <Sequence Name="042_Fire_Temple"/>
+    <Sequence Name="043_Anticipating_Treasure"/>
+    <Sequence Name="044_Forest_Temple"/>
+    <Sequence Name="045_Hyrule_Castle_Courtyard"/>
+    <Sequence Name="046_Ganondorf_Theme_Organ"/>
+    <Sequence Name="047_Lon_Lon_Ranch"/>
+    <Sequence Name="048_Goron_City"/>
+    <Sequence Name="049_Hyrule_Field_Morning"/>
+    <Sequence Name="050_Got_Spiritual_Stone"/>
+    <Sequence Name="051_Bolero_of_Fire"/>
+    <Sequence Name="052_Minuet_of_Forest"/>
+    <Sequence Name="053_Serenade_of_Water"/>
+    <Sequence Name="054_Requiem_of_Spirit"/>
+    <Sequence Name="055_Nocturne_of_Shadow"/>
+    <Sequence Name="056_Mini_Boss_Battle"/>
+    <Sequence Name="057_Got_Item"/>
+    <Sequence Name="058_Temple_of_Time"/>
+    <Sequence Name="059_Rescued_Epona"/>
+    <Sequence Name="060_Kokiri_Forest"/>
+    <Sequence Name="061_Got_Fairy_Ocarina"/>
+    <Sequence Name="062_Lost_Woods"/>
+    <Sequence Name="063_Spirit_Temple"/>
+    <Sequence Name="064_Horse_Minigame"/>
+    <Sequence Name="065_Horse_Minigame_Goal"/>
+    <Sequence Name="066_Ingos_Theme"/>
+    <Sequence Name="067_Got_Medallion"/>
+    <Sequence Name="068_Sarias_Song"/>
+    <Sequence Name="069_Eponas_Song"/>
+    <Sequence Name="070_Zeldas_Lullaby"/>
+    <Sequence Name="071_Suns_Song"/>
+    <Sequence Name="072_Song_of_Time"/>
+    <Sequence Name="073_Song_of_Storms"/>
+    <Sequence Name="074_Navi_in_Flight"/>
+    <Sequence Name="075_Deku_Tree_Theme"/>
+    <Sequence Name="076_Windmill"/>
+    <Sequence Name="077_Legends_of_Hyrule"/>
+    <Sequence Name="078_Game_Shops"/>
+    <Sequence Name="079_Sheiks_Theme"/>
+    <Sequence Name="080_Zoras_Domain"/>
+    <Sequence Name="081_Zelda_Sting"/>
+    <Sequence Name="082_Zeldas_Theme_Ending"/>
+    <Sequence Name="083_Got_Master_Sword"/>
+    <Sequence Name="084_Ganondorfs_Theme"/>
+    <Sequence Name="085_Shop"/>
+    <Sequence Name="086_Chamber_of_Sages"/>
+    <Sequence Name="087_File_Select" Extract="false"/>
```

Comment:
I wonder if you could add a comment above this stating this data is a pointer to `040_Fairy_Fountain` instead of something inside the repo

Reply:
👍